bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/WebrevStorage.java

```diff
@@ -141,6 +145,33 @@ private static void clearDirectory(Path directory) {
         }
     }
 
+    private void awaitPublication(URI uri, Duration timeout) throws IOException {
+        var end = Instant.now().plus(timeout);
+        var uriBuilder = URIBuilder.base(uri);
+        var client = HttpClient.newBuilder()
+                               .connectTimeout(Duration.ofSeconds(30))
+                               .build();
+        while (Instant.now().isBefore(end)) {
+            var uncachedUri = uriBuilder.setQuery(Map.of("nocache", UUID.randomUUID().toString())).build();
+            var request = HttpRequest.newBuilder(uncachedUri)
+                                     .timeout(Duration.ofSeconds(30))
+                                     .GET()
+                                     .build();
+            try {
+                var response = client.send(request, HttpResponse.BodyHandlers.ofString());
+                if (response.statusCode() < 400) {
```

Comment:
```suggestion
                if (respons.statusCode() >= 200 && response.statusCode() < 400) {
```

Reply:
I don't think < 200's propagate outside of the http client implementation, so should be reasonably safe.